components/MessageCard.vue

```diff
@@ -15,10 +15,57 @@
 <script>
 export default {
   props: ["message", "isThread"],
+
   methods: {
     showThread() {
       this.$emit('showThread', this.message.ts);
     },
-  }
+    replaceLinkWithATag(actualLink, displayLink){
+      var htmlTagLink = actualLink.link(actualLink);
+      if(displayLink){
+        htmlTagLink = displayLink.link(actualLink);
+      }
+      htmlTagLink = htmlTagLink.replace('a href=', 'a target="_blank" href=');
+      return htmlTagLink
+    },
+    escape(text, links){
+      var surroundedByAngleBracket =
+        text.match(/<.+?\>/g).filter(n => !links.includes(n));
+      surroundedByAngleBracket.forEach((elementInside) => {
+        text = text.replace(elementInside, elementInside.replace(/</g,'&lt;').replace(/>/g,'&gt;'));
+      });
+      return text;
+    },
+    messageWithHTMLTag(text) {
+      var matchedLinks = text.match(/<http.+?\>/g);
+      var imageTags = [];
```

Comment:
変数名はわかりやすくてセンスがいいですね 👍🏻 
letで再宣言を禁止した変数宣言の方が良いです。
```suggestion
      let imageTags = [];
```

Reply:
修正しました。https://github.com/takuuuu517/nuxt-sample-app/pull/3/commits/badc495552ba363893f5639d9251673986affc1f